Readme Azure Spring Cloud.md

```diff
@@ -0,0 +1,152 @@
+cd spring-petclinic-rest
+./mvnw spring-boot:run
+```
+
+### With Docker
+```
+docker run -p 9966:9966 springcommunity/spring-petclinic-rest
+```
+
+You can then access petclinic here: [http://localhost:9966/petclinic/](http://localhost:9966/petclinic/)
+
+There are actuator health check and info routes as well: 
+* [http://localhost:9966/petclinic/actuator/health](http://localhost:9966/petclinic/actuator/health)
+* [http://localhost:9966/petclinic/actuator/info](http://localhost:9966/petclinic/actuator/info)
+
+## Running Spring Petclinic in Azure Spring Cloud
+
+### Install the Azure CLI extension
+
+Install the Azure Spring Cloud extension for the Azure CLI using the following command
+
+```bash
+    az extension add --name spring-cloud
+```
+Note - `spring-cloud` CLI extension `2.1.0` or later is a pre-requisite to enable the
+latest Java in-process agent for Application Insights. If you already 
+have the CLI extension, you may need to upgrade to the latest using --
+
+```bash
+    az extension update --name spring-cloud
+```
+### Provision Azure Spring Cloud service instance using Azure CLI
+
+### Prepare your environment for deployments
+Set the following environment variables required in the Azure Spring Cloud deployment below.
+
+```bash
+
+    export SUBSCRIPTION=subscription-id # customize this
+    ...
+    export RESOURCE_GROUP=resource-group-name # customize this
+    ...
+    export REGION=westus2 # customize this
+    ...
+    export SPRING_CLOUD_SERVICE=azure-spring-cloud-name # customize this
+    ...
+```
+### Login to Azure 
+Login to the Azure CLI and choose your active subscription. Be sure to choose the active subscription that is whitelisted for Azure Spring Cloud
+
+```bash
+    az login
+    az account list -o table
+    az account set --subscription ${SUBSCRIPTION}
+```
+
+### Create Azure Spring Cloud service instance
+Prepare a name for your Azure Spring Cloud service.  The name must be between 4 and 32 characters long and can contain only lowercase letters, numbers, and hyphens.  The first character of the service name must be a letter and the last character must be either a letter or a number.
+
+Create a resource group to contain your Azure Spring Cloud service.
+
+```bash
+    az group create --name ${RESOURCE_GROUP} \
+        --location ${REGION}
+```
+
+### Create an instance of Azure Spring Cloud.
+
+```bash
+    az spring-cloud create --name ${SPRING_CLOUD_SERVICE} \
+        --sku standard --enable-java-agent \
+        --resource-group ${RESOURCE_GROUP} \
+        --location ${REGION}
+```
+
+The service instance will take around five minutes to deploy.
+
+Set your default resource group name and cluster name using the following commands:
+
+```bash
+    az configure --defaults \
+        group=${RESOURCE_GROUP} \
+        location=${REGION} \
+        spring-cloud=${SPRING_CLOUD_SERVICE}
+```
+
+### Create microservice applications
+
+Create 5 microservice apps.
```

Comment:
Create a microservice app (instead of 5)

Reply:
Typo. Will fix it.